packages/e2e/test/web-extension/wdio.conf.js

```diff
@@ -50,6 +50,6 @@ module.exports = {
             'chromedriver'
         ],
         specs: ['./**/*.spec.ts'],
-        waitforTimeout: 10000
+        waitforTimeout: 30000
```

Comment:
This doesn't belong in the same commit as the CML replacement

Reply:
The commit message is missing some context and includes redundant information already expressed in the diff. Please have another go at this commit message, with an explanation of why this is needed.